### PR TITLE
🎉 Destination databricks: move driver downloading to gradle

### PR DESCRIPTION
**Pull request by tuliren**
## What
- Currently the Databricks JDBC driver is downloaded in CI, which is not only hacky, but it does not always work. For example, the automated daily connector integration test never works for Databricks.
- This PR moves the driver downloading process to the dockerfile.

## Recommended reading order
1. `Dockerfile`
2. `README.md` & `databricks.md`
3. The rest.

## 🚨 User Impact 🚨
None expected.

### CODE REVIEW

**Comment by tuliren** [comment]
Actually this does not work. Need to download the driver before compiling the code.

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/destination-databricks/Dockerfile [hunk 1]
```diff
@@ -7,6 +21,22 @@ ENV APPLICATION destination-databricks
 COPY build/distributions/${APPLICATION}*.tar ${APPLICATION}.tar
 
 RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
+RUN if [ -f "lib/SparkJDBC42.jar" ]; then \
+    echo "[Databricks] JDBC driver exists"; \
+  else \
+    echo "[Databricks] JDBC drier does not exist"; \
```
Comment (ChristopheDuong) [inline airbyte-integrations/connectors/destination-databricks/Dockerfile:27]:
```suggestion
    echo "[Databricks] JDBC driver does not exist"; \
```

Comment (tuliren) [reply airbyte-integrations/connectors/destination-databricks/Dockerfile]: Thanks.

**Pull request merged**